lib/plugins/aws/package/compile/events/cloud-front.js

```diff
@@ -612,6 +612,7 @@ class AwsCompileCloudFrontEvents {
 
       Object.assign(Resources, { [this.cloudFrontDistributionLogicalId]: CloudFrontDistribution });
 
+      // #7747, could not replace _.merge with Object.assign, as this does deep merges
```

Comment:
I think these assignments look pretty safe to be refactored - do you think we should keep merge here?